subnet.go

```diff
@@ -127,16 +121,6 @@ func (s *subnet) FanOverlay() string {
 	return s.FanOverlay_
 }
 
-// AllocatableIPHigh implements Subnet.
```

Comment:
These are the deprecated ones that juju is not using anymore, right?

Reply:
Yes, these haven't been used in juju 2.x that I can find.  Definitely not in the tip of any 2.x release.